test/integration/test_remote_files.py

```diff
@@ -115,6 +118,41 @@ def test_fetch_from_import(self):
 
         assert os.path.exists(os.path.join(self.library_dir, "a"))
 
+    def test_fetch_from_drs(self):
+        CONTENT = "a\n"
+        history_id = self.dataset_populator.new_history()
+
+        def create_drs_object():
+            hda = self.dataset_populator.new_dataset(history_id, content=CONTENT, wait=True)
+            # Force the md5 hash to be evaluated. Otherwise, the DRS endpoint will attempt to dispatch an md5
+            # task and returns a 204 Try Later. This will cause a timeout in a test environment without multiple
```

Comment:
Wait, are you saying we have a celery task calling another celery task ? That would be quite bad.

Reply:
there's no way to wait for the task to complete, just submitting the request again will fetch the dataset again ? that's not so great, but ok, this is fine for now.